docs/api/ReactWrapper/text.md

```diff
@@ -21,8 +21,8 @@ expect(wrapper.text()).to.equal('important');
 ```
 
 ```jsx
-const wrapper = mount(<div><Foo /><b>important</b></div>);
-expect(wrapper.text()).to.equal('<Foo />important');
+const wrapper = mount(<div><Foo>This is <b>really</b> important</Foo></div>);
```

Comment:
sorry for the confusion; the children you pass into `Foo` are irrelevant; it matters what the `Foo` component *itself* renders.

I was envisioning something like
```jsx
function Foo() {
  return <div>this is a foo</div>
}
```